### ISSUE TITLE
Cannot Request Movie on Mobile Browser

### ISSUE THREAD

**Issue by jstevenson72** [Ombi-app/Ombi#4062]
**Describe the bug**
The new Request UI does not render the same on Mobile as on Desktop.  And the Text obscures the entire clickable area of the poster, and is so large the button to request is not there.  Also there is no area that can be clicked to bring you to the details to request it there.

On Desktop there is only a small area clickable to get to the details if there even is room, I sometimes have to reduce the scale to 90% in browser to see the button.

**To Reproduce**
1. Search for Cosmic Sin.
2. Click on the Search Result.
3. Text obscures image and you cannot request it at all on mobile.

**Expected behavior**
I would expect that on mobile we dont show the text, or at least allow clicking the text to take you to the details page to see the full UI for requesting.

I think you may have fixed the clicking text issue on desktop.

**Screenshots**
![image](https://user-images.githubusercontent.com/1573442/108602858-1d494200-736a-11eb-81bc-2805548f98ff.png)

**Logs (Logs directory where Ombi is located)**
If applicable, a snippet of the logs that seems relevant to the bug if present.

**Desktop (please complete the following information):**
 - OS: Android 11 / Windows 10

**Ombi Version (please complete the following information):**
 - Version **4.0.1116.0**
- Media Server Plex
- Database Type: SQLite

**Additional context**

**Comment by tidusjar** [comment]
@twanariens something you could look into?

**Comment by twanariens** [comment]
I think I am missing something here... @jstevenson72 

When I try to reproduce, I can't see the text / request button because this is activated on hover (which does not work on Mobile since it is a touch display). When I click on the search result, it opens the Movie Details page.

I am able to reproduce when I resize my Chrome window.. which will delete the summary when the width of the screen reaches 400px.

So I am trying to figure out how you are getting what you screenshotted cause I am in the dark right now.

**Comment by jstevenson72** [comment]
The easiest way to repro the issue is on a phone.  Open chrome on mobile
and head to a site, do a search.  The poster image that you get doesn't
obviously have a hover option, but when you click, it produces the hover
effect.  Which means you cannot click the poster.  And the text forces the
button off screen.

**Comment by twanariens** [comment]
Okay I have seem to "reproduce" the issue, but it is still working as intended meaning:
The request button is shown on the bottom of the poster (overlaying the text). The text is clickable so it will redirect you to the Details page.

Can you give me more information on the device you are using (so I can check the resolution and see if I can replicate).

**Comment by jstevenson72** [comment]
> Okay I have seem to "reproduce" the issue, but it is still working as intended meaning:
> The request button is shown on the bottom of the poster (overlaying the text). The text is clickable so it will redirect you to the Details page.
> 
> Can you give me more information on the device you are using (so I can check the resolution and see if I can replicate).

Sure, it is a Pixel 5, running Chrome.  But my whole family (various phones) has experienced the same issue.

Chrome v88.0.4324.181

**Comment by twanariens** [comment]
![testing2](https://user-images.githubusercontent.com/34845004/108607140-ddf91080-73be-11eb-8ff6-49dddda6f994.gif)

Just to show you I am not lying. Hahahaha.

I will work out the poster issue by defining a max poster size. I will let you know when the new build will contain that fix. I do need your help with testing it since I am unable to test it myself (unfortunately).

**Comment by jstevenson72** [comment]
Looks like your using the Developer Tools / Device Option in Chrome so that
you get a drag bar on a mobile browser, instead of an actual mobile
browser?  If you are simulating this on the desktop, I also cannot
reproduce the issue.  Clicking once will take you directly to the details
page as you intend.

However doing the exact same action, actually on an android Pixel phone,
using Chrome, you get the screenshot (right side) I attached.

I just tested this on an iPhone 8 on Safari, and it too works fine.

So the problem must be limited to Android/Chrome.

Thanks for looking into it either way.  Let me know if you would like me to
test anything.
